### ISSUE TITLE
Incorrect bumped version for v0.8.16

### ISSUE THREAD

**Issue by aherranz** [ChicagoBoss/ChicagoBoss#607]
```bash
angel@Lenovo-X301: /ChicagoBoss ((v0.8.16))$ rgrep 0\.8\.15 .
./src/boss.app.src:    {vsn, "0.8.15"},
./skel/rebar.config:  {boss, ".*", {git, "git://github.com/ChicagoBoss/ChicagoBoss.git", {tag, "v0.8.15"}}}
./ebin/boss.app:              {vsn,"0.8.15"},
./rebar.config:        {tinymq,                ".*",   {git, "git://github.com/ChicagoBoss/tinymq.git",        {tag, "v0.8.15"}}},
./rebar.config:        %{tiny_pq,               ".*",   {git, "git://github.com/ChicagoBoss/tiny_pq.git",       {tag, "v0.8.15"}}},
```

**Comment by danikp** [comment]
I will put more attention to *.app.src files before making release.

Regarding other libraries: their version may not match CB version, although I try to keep them synced. But I will not make a release without  any commit just to bump version.

**Comment by aherranz** [comment]
Nice!!!

I think this also solves the issue #610.

**Comment by aherranz** [comment]
Solved in 0.8.19.
